app/views/shared/_problem_order_details.html.haml

```diff
@@ -1,15 +1,17 @@
 %table.table.table-striped.table-hover
   %thead
     %tr.borderless
-      %td{colspan: 6}
-      %td= link_to t(".assign_price_policies"),
-        assign_price_policies_path,
-        method: :post,
-        class: %w(btn btn-primary)
+      %td{colspan: 8}
+        .text-right
+          = link_to t(".assign_price_policies"),
+            assign_price_policies_path,
+            method: :post,
+            class: %w(btn btn-primary)
     %tr
       %th= Order.model_name.human
       %th= OrderDetail.model_name.human
-      %th= Order.human_attribute_name(:ordered_at)
+      %th= problem_order_date_header(local_assigns[:reservation_list_page])
```

Comment:
This variable name is a little confusing. It almost seems like it's related to pagination.